src/internal.c

```diff
@@ -24827,6 +24832,8 @@ int SendCertificateRequest(WOLFSSL* ssl)
 
     return ret;
 }
+#endif /* NO_TLS */
```

Comment:
`!NO_TLS`

Reply:
Good catch. Corrected that one & applied some other edits, added closing gate comments, etc.

Note added file: [src/tls13.c](https://github.com/wolfSSL/wolfssl/pull/8514/files#diff-927d62a3fce45ef1b7d9ba70df049ab4dc21ea53fc2491b5c006fc96c5af680e)